client/my-sites/checkout/composite-checkout/payment-method-processors.js

```diff
@@ -123,6 +125,45 @@ export async function stripeCardProcessor(
 	return pending;
 }
 
+export async function ebanxCardProcessor( submitData ) {
```

Comment:
Since #44533 this will need to read `ebanxCardProcessor(submitData, {includeDomainDetails, includeGSuiteDetails})`